spec/models/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe User do
+
+  subject { create :user }
+
+  it { should have_many :expenses }
+
+  it { should have_many :monthly_bills }
+
+  it { should validate_presence_of :first_name }
+
+  it { should validate_presence_of :last_name }
+
+  it 'generates a display name when created' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.